src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -7,6 +7,7 @@ method java.lang.CharSequence length
 method java.lang.Comparable compareTo java.lang.Object
 method java.lang.Object getClass
 method java.lang.Object equals java.lang.Object
+method java.lang.Object getClass
```

Comment:
And what do you expect to do with the `Class` that would be safe? Generally anything smelling of reflection should be avoided unless it has been analyzed carefully.